packages/code-studio/src/main/AppInit.tsx

```diff
@@ -243,7 +247,24 @@ const AppInit = (props: AppInitProps) => {
         links: data.links,
       };
 
+      const configs = await coreClient.getServerConfigValues();
+
+      const serverConfig: ServerConfigValues = {};
+
+      for (let i = 0; i < configs.length; i += 1) {
+        const [config, version] = configs[i];
+
+        if (config === 'java.version') {
+          serverConfig.javaVersion = version;
+        } else if (config === 'deephaven.version') {
+          serverConfig.deephavenVersion = version;
+        } else if (config === 'barrage.version') {
+          serverConfig.barrageVersion = version;
+        }
+      }
```

Comment:
I think instead of building the server config values into our own object, we should just create a Map, which would eliminate most of this code, e.g. `const serverConfig = new Map(configs);`
Then have a few constants for the settings that we know about. This has a couple of advantages:
1) We don't need to change this part for each new settings
2) We could pass the server config to plugins, and they can read a server config value that we may not know/care about.